configuration.md

```diff
@@ -36,110 +36,110 @@ As part of Mosets Tree's [multi-directory]({{version}}/multi-directory) system,
  This is where all the notification e-mails are sent to. You can enter more than one e-mail by separating them with commas.
 
  ### Adding new listing requires approval
- Set `Yes` if you want all newly submitted listings from front-end to go through approval before it is published. Listings that are pending for approval can be reviewed in Mosets Tree's back-end:
+ Set `Yes` to require admin/manager approval of all newly submitted listings from the site front-end before they are published. Listings that are pending approval can be reviewed in the Mosets Tree's back-end:
 
   "**Mosets Tree -> Pending Approval -> Listings**"
 
  ### Number of days to show as new listing
- A "New" badge will be shown beside listing's name if it is a recently added listing. This config lets you to configure the number of days you want a listing to be displayed as new listing.
+ A "New" badge/icon will be shown next to a listing's title/name if it is a recently added listing. This configuration allows you to set the number of days that a listing will be displayed with the new listing badge/icon.
```

Comment:
re: badge/icon
I think we should call it 'badge'. Icon to me is a graphical representation of something. Here's we are showing a text (ie: 'NEW') with an attention grabbing background.

re: title/name
Title I think we should reserve for browser title, ie: the text that goes between <title> ... </title>. Just using name here I think is fine.

Thoughts?